app/assets/stylesheets/pages/about.scss

```diff
@@ -73,8 +126,12 @@
         position: relative;
 
         .member-description {
+          .name {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets